runtime/src/main/java/org/corfudb/runtime/collections/LrMultiStreamMergeStreamListener.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.runtime.collections;
+
+/**
+ * This is the callback interface that any client subscribing to CorfuStore updates must implement.
+ *
+ * This interface sees ordered logs from :
+ * 1. client-streams from client-Namespace, and,
+ * 2. LrStatusTable from corfuSystem-Namespace.
+ */
+public interface LrMultiStreamMergeStreamListener extends StreamListener{
```

Comment:
I suppose this interface can be removed as it is not doing anything different from the StreamListener

Reply:
added LRMultiNamespaceListener as the template